### DISCUSSION TITLE
I use Quarto for my personal website and Bear app for new content

### DISCUSSION THREAD

**Discussion by beyond2060** [quarto-dev/quarto-cli#9994] [Show and Tell]
### Description

I use the Bear app as my Personal Knowledge Management system, and Quarto for my website. The two work together pretty well. The editing experience in Bear is great, and it's simple to export to markdown, including images. I've been looking for a long time for a way to combine my PKM with my public website (including using Quarto for my PKM itself, which didn't really work), and I think using Bear for the PKM and Quarto for the public site is a great solution. [I wrote about it here](https://www.beyond2060.com/technical/24-06/My-Personal-Knowledge-Management-System.html)
https://www.beyond2060.com/technical/24-06/My-Personal-Knowledge-Management-System.html

**Comment by mcanouil** [comment]
Thanks for sharing!
